### PR TITLE
Fixes #8909:  missing rudder_expected_reports.csv.res when starting the agent for the first time after an update - technique changes

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8909

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: techniques/system/common/1.0/promises.st [hunk 1]
```diff
@@ -656,6 +656,12 @@ bundle agent garbage_collection
         comment     => "Rotate file if above specified size",
         rename      => rotate("10"),
         file_select => bigger_than("1M");
+
+  methods:
+
+       # Directly call the cleanup bundle from ncf's logger_rudder
```
Comment (jooooooon) [inline techniques/system/common/1.0/promises.st]: You may as well call this log_rudder, because that's it's future name

**Comment by jooooooon** [comment]
This depends on Normation/ncf#423, don't merge yet

File: techniques/system/common/1.0/promises.st (hunk 1 above)
Comment (amousset) [inline techniques/system/common/1.0/promises.st]: Yes, it should be done for 3.2+

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**